src/test/kotlin/ar/edu/unahur/obj2/semillasAlViento/SemillasAlVientoTest.kt

```diff
@@ -0,0 +1,123 @@
+package ar.edu.unahur.obj2.semillasAlViento
+
+import io.kotest.assertions.throwables.shouldThrowAny
+import io.kotest.core.spec.style.DescribeSpec
+import io.kotest.matchers.booleans.shouldBeFalse
+import io.kotest.matchers.booleans.shouldBeTrue
+import io.kotest.matchers.booleans.shouldNotBeFalse
+import io.kotest.matchers.collections.shouldContainExactly
+import io.kotest.matchers.shouldBe
+
+
+class SemillasAlVientoTest : DescribeSpec({
+
+    describe("Huerta"){
+
+        val mentaA = Menta(2000,0.3f)
+        val mentaB = Menta(2000,0.5f)
+
+        // Se tuvo que sacar el atributo Boolean porque la Soja transgénica fue diferenciada en una nueva clase
+        val sojaA = Soja(2006,0.4f)
+        val sojaB = Soja(2008,0.8f)
+        val sojaC = Soja(2009,1.2f)
+        // Se cambió de clase Soja a Soja transgénica.
+        val sojaTransgenica = SojaTransgenica(2008,1.2f)
+
+        val parcelaA = Parcela(8,10,5)
+        val parcelaB = Parcela(20,10,9)
+        val parcelaC = Parcela(20,10,7) // Se agrega para poder realizar el test de agricultoras
+
+        val agricultora = Agricultora(listOf(parcelaA, parcelaC) as MutableList<Parcela>) // Se cambia la parcela B por la C
```

Comment:
Me olvidé de comentartelo: hacer un casteo así es fulero, mejor directamente crear una lista mutable.
```suggestion
        val agricultora = Agricultora(mutableListOf(parcelaA, parcelaC)) // Se cambia la parcela B por la C
```